README.md

```diff
@@ -369,6 +375,40 @@ For example:
 The [Laravel Transmorpher Client](https://github.com/cybex-gmbh/laravel-transmorpher-client) will receive this information and store it. It can also create URLs with
 transformations.
 
+## PDF handling
+
+Requesting a PDF file will return the document.
+Metadata can be removed optionally by setting the `.env` key: 
+
+```dotenv
+TRANSMORPHER_DOCUMENT_REMOVE_METADATA=true
+```
+
+### Images
+
+When an image format transformation is specified, an image of a page will be returned.
+
+All available image transformations can also be applied to PDF image derivatives.
+Requesting a PDF also follows the same URL structure as images, just replace `images` with `documents`.
+
+The page which should be used as image can be specified with the `p` transformation.
+Otherwise, the first page will be used.
+
+Additionally, the pixels per inch can be specified with the `ppi` transformation.
+This will define the resolution of the image generated from the PDF.
+By default, 300 ppi is used.
+Use the `.env` key to specify another default:
+
+```dotenv
+TRANSMORPHER_DOCUMENT_DEFAULT_PPI=100
+```
```

Comment:
same as in .env.example probably. or even explain what it does and what would happen if you changed it to a lower or higher value. also, resolution typically defines the total amount of pixels on the according axis. ppi is basically pixel density.

depending on the background and reading comprehension, someone might expect an image with 300 pixels on whatever axis, but not an image with a resolution of 3000 pixels, which is absolutely would be a possible result.

Reply:
Adjusted the default value and changed the wording.